src/plugins/htmlVideoPlayer/style.scss

```diff
@@ -74,6 +74,16 @@ video[controls]::-webkit-media-controls {
     display: inline-block;
 }
 
+.videoSecondarySubtitlesInner {
+    max-width: 70%;
+    background-color: rgba(0, 0, 0, 0.8);
+    margin: auto;
+    display: block;
```

Comment:
This is some great inside knowledge that I wouldn't have found out otherwise.
I went ahead and made these changes. However, I was a bit confused because this diff places the padding options inside `video[controls]::-webkit-media-controls`, when they are actually  in `.videoSubtitles`.

Either way the subtitles look good with flexbox.

Reply:
> when they are actually in `.videoSubtitles`.

This is right - it should go into `.videoSubtitles` when using `patch -p1 < patchfile`.

`@@ -65,20 +65,19 @@ video[controls]::-webkit-media-controls {` was genered by `git diff` :man_shrugging: